CMake/options.cmake

```diff
@@ -203,9 +205,13 @@ if (UNIX)
     pkg_check_modules(DBUS dbus-1)
     if (NOT(DBUS_FOUND AND WLDCLIENT_FOUND AND WLDCURSOR_FOUND AND WLDPROTO_FOUND AND XKBCOMMON_FOUND))
       message (STATUS "Not all software modules 'wayland-client wayland-cursor wayland-protocols xkbcommon dbus-1' are present")
-      message (STATUS "Consequently, building for the Wayland backend is not possible.")
-      message (FATAL_ERROR "*** Aborting ***")
+      message (STATUS "Consequently, OPTION_USE_WAYLAND is set to OFF.")
+      unset (OPTION_USE_WAYLAND CACHE)
+      set (OPTION_USE_WAYLAND 0)
     endif (NOT(DBUS_FOUND AND WLDCLIENT_FOUND AND WLDCURSOR_FOUND AND WLDPROTO_FOUND AND XKBCOMMON_FOUND))
+  endif (OPTION_USE_WAYLAND)
+    
+  if (OPTION_USE_WAYLAND)
     option (OPTION_WAYLAND_ONLY "support Wayland backend only" OFF)
     set (FLTK_USE_WAYLAND 1)
     if (NOT OPTION_WAYLAND_ONLY)
```

Comment:
Does line 215 setting OPTION_WAYLAND_ONLY to OFF mean that this line (217) can never be true?

Reply:
Line 215 does not set the option to off. It creates the option and its associated tooltip text, and declares its default value is OFF. The user can then turn it on or off, and line 217 controls what happens next.